lib/fs.js

```diff
@@ -1921,79 +1921,6 @@ WriteStream.prototype.close = ReadStream.prototype.close;
 // There is no shutdown() for files.
 WriteStream.prototype.destroySoon = WriteStream.prototype.end;
 
-
-// SyncWriteStream is internal. DO NOT USE.
-// Temporary hack for process.stdout and process.stderr when piped to files.
-function SyncWriteStream(fd, options) {
-  Stream.call(this);
-
-  options = options || {};
-
-  this.fd = fd;
-  this.writable = true;
-  this.readable = false;
-  this.autoClose = options.autoClose === undefined ? true : options.autoClose;
-}
-
-util.inherits(SyncWriteStream, Stream);
-
-
-// Export
-Object.defineProperty(fs, 'SyncWriteStream', {
-  configurable: true,
-  writable: true,
-  value: SyncWriteStream
-});
```

Comment:
I have to wonder if it isn't actually useful in those cases? Why would someone ever use this if something didn't specifically need it? idk..